docs/user-guide/how_to_use_pytorch_plugin.md

```diff
@@ -0,0 +1,60 @@
+# Pytorch Plugin User Guide
+
+## Introduction
+
+**Pytorch plugin** is designed to optimize the user experience when running pytorch jobs, it not only allows users to write less yaml, but also ensures the normal operation of Pytorch jobs.
+
+## How the Pytorch Plugin Works
+
+The Pytorch Plugin will do three things:
+
+* Open ports used by Pytorch for all containers of the job
+* Force open `svc` plugins
+* Add some envs such like `MASTER_ADDR`, `MASTER_PORT`, `WORLD_SIZE`, `RANK` which pytorch distributed training needed to containers automatically
+
+## Parameters of the Pytorch Plugin
+
+### Arguments
+
+| ID   | Name   | Type   | Default Value | Required | Description                        | Example            |
+| ---- | ------ | ------ | ------------- | -------- | ---------------------------------- | ------------------ |
+| 1    | master | string | master        | No       | Name of Pytorch master             | --master=master    |
+| 2    | worker | string | worker        | No       | Name of Pytorch worker             | --worker=worker    |
+| 3    | port   | string | 23456         | No       | The port to open for the container | --port=23456       |
+
+## Examples
+
+```yaml
```

Comment:
Excuse me, have you tested the job yaml locally?

Reply:
yes, this is the test result:

![image](https://user-images.githubusercontent.com/88617921/179649649-00008225-d4ad-4456-a739-2b3966db06b4.png)